scripts/utils/index.js

```diff
@@ -35,13 +39,13 @@ const getConnex = async (network) => {
 }
 
 const getTimeConstraints = () => {
-  const end = dayjs().endOf('month').utc().unix();
+  const end = dayjs().add(1, 'month').endOf('month').subtract(4, 'hours').unix();
```

Comment:
This needs to be cleaned up before merge. Can we get rid of dayjs in favour of something like: https://stackoverflow.com/a/52229801